### ISSUE TITLE
Change Address modal triggers horizontal scrolling

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#20382]
Does not meet the success criteria for [SC 1.4.10](https://www.w3.org/WAI/WCAG22/Understanding/reflow)

<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/7c8635ff-217d-47d8-a17e-8e28256064aa" />

### How to reproduce

In Firefox go to Preferences > General > Zoom
- Ensure default Zoom is set to 100%
- check Zoom text only

Create shop with 1 product

Visit storefront homepage
- Set viewport to 320px wide
- Zoom text to 200%
- Checkout with product
   - On checkout confirm page click 'Change shipping address'
